src/view/show_mode/editor/show_ui_widgets/button_list_with_submit_value.py

```diff
@@ -0,0 +1,62 @@
+from PySide6.QtWidgets import (QWidget, QHBoxLayout, QPushButton, QVBoxLayout, QListWidget,
+                               QLineEdit, QLabel, QDoubleSpinBox)
+
+from model import UIWidget, UIPage, Filter
+from model.filter import FilterTypeEnumeration
+from view.show_mode.editor.show_ui_widgets.constant_button_list import ConstantNumberButtonList
+
+
+class ButtonsWithValueSubmit(UIWidget):
+    """ UI widget for the showmode (extended with ConstantNumberButtonList also for the editor mode)
+    Provides a textfield with a submit button to update a value (of a constant node),
+    provides also (from ConstantNumberButtonList) a button list to send pre-defined values
+    """
```

Comment:
i'm not sure about where to write which type of doc, should it explain how it works, or how to describe the class?

Reply:
What the class is actually about and how to use it. What you've written is actually fine.